Source/Renderer/Buffer.js

```diff
@@ -7,6 +7,7 @@ define([
         '../Core/DeveloperError',
         '../Core/IndexDatatype',
         '../Core/WebGLConstants',
+        '../Core/Check',
```

Comment:
Make sure that you add the define paths in alphabetical order. Same applies to other files.